nav2_rotation_shim_controller/src/nav2_rotation_shim_controller.cpp

```diff
@@ -141,6 +158,8 @@ void RotationShimController::deactivate()
     node->remove_on_set_parameters_callback(dyn_params_handler_.get());
   }
   dyn_params_handler_.reset();
+
+  last_angular_vel_ = std::numeric_limits<double>::max();
```

Comment:
I think putting into activate makes more sense so that the first initialization also has it reset (I know the header has it set on construction as this as well, but I don't like relying on that kind of behavior since we have alot of hands in the codebase)